src/gui/accountsettings.cpp

```diff
@@ -333,6 +334,48 @@ void AccountSettings::slotEditCurrentIgnoredFiles()
     openIgnoredFilesDialog(f->path());
 }
 
+void AccountSettings::slotOpenMakeFolderDialog()
+{
+    const auto &selected = _ui->_folderList->selectionModel()->currentIndex();
+
+    if (!selected.isValid()) {
+        qCWarning(lcAccountSettings) << "Selection model current folder index is not valid.";
+        return;
+    }
+
+    const auto &classification = _model->classify(selected);
+
+    if (classification != FolderStatusModel::SubFolder && classification != FolderStatusModel::RootFolder) {
+        return;
+    }
+
+    const QString fileName = [this, &selected, &classification] () {
```

Comment:
Don't necessarily go change it, it is more a "by the way for next time" comment. So you know you can omit the parenthesis for lambdas not taking any parameter. It's a tiny bit less syntactic noise which is always welcome.